lib/jekyll/strapi/collection.rb

```diff
@@ -20,27 +20,35 @@ def generate?
       def each
         # Initialize the HTTP query
         path = "/#{@config['type'] || @collection_name}?_limit=10000"
-        uri = URI("#{@site.endpoint}#{path}")
-        Jekyll.logger.info "Jekyll Strapi:", "Fetching entries from #{uri}"
+        uri = URI("#{@site.endpoint}/api#{path}")
         # Get entries
-        response = Net::HTTP.get_response(uri)
+        Jekyll.logger.info "Jekyll Strapi:", "Fetching entries from #{uri}"
+        # response = Net::HTTP.get_response(uri)
+        response = strapi_request(uri)
         # Check response code
         if response.code == "200"
-          result = JSON.parse(response.body, object_class: OpenStruct)
```

Comment:
Why add an extra variable to store data; wasn't    ```result = JSON.parse(response.body, object_class: OpenStruct)```

Reply:
To avoid lookups to data.  I think this is visible in below parts, which are not attached to this comment.